samples/azure-service-bus-netstandard/native-integration-pub-sub/ASBS_1/NativeSubscriberA/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.ServiceBus;
+using NativeSender;
+class Program
+{
+    static string EnclosedMessageTypesHeader = "NServiceBus.EnclosedMessageTypes";
+
+    static string ConnectionString
+    {
+        get
+        {
+            var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus_ConnectionString");
+            if (string.IsNullOrWhiteSpace(connectionString))
+            {
+                throw new Exception("Could not read the 'AzureServiceBus_ConnectionString' environment variable. Check the sample prerequisites.");
+            }
+
+            return connectionString;
+        }
+    }
+
+    static async Task Main()
+    {
+        var subscriptionName = "Samples.ASB.NativeIntegration.NativeSubscriberA";
+        
+        Console.Title = subscriptionName;
+
+
+        await TopologyManager.CreateSubscription(
+            ConnectionString,
+            subscriptionName,
+        #region SubscriberAFilter
```

Comment:
Similar to this, I'd suggest adding a snippet for a mandatory rule name and elaborate why is it needed (see my previous comment).